### DISCUSSION TITLE
Halmet sample code query

### DISCUSSION THREAD

**Discussion by ESchlotfeldt** [hatlabs/discussions#128] [Q&A] [answered]
Hello Matti - I'm trying to set up a number of I2C devices (BME280, INA226, ADAfruit 9DoF board) on my Halmet. Previously got them all to work on an ESP-32 Dev board using the standard Arduino wire.h library. I see you've used the TwoWire.h library to drive the OLED display in your sample code. I'm struggling to get the other I2C devices to work using TwoWire.h as their libraries seem to be referencing and using wire.h. Is there an easy way to convert your OLED display code to using wire.h? Or is there another (simple!) way of resolving this? Thanks!

**Comment by mairas** [comment, marked as answer]
It's the same library. If you press F12 with the cursor on top of the `TwoWire` class name, you'll see that the `TwoWire` class is defined in `Wire.h`.

**Reply by ESchlotfeldt** [reply]
Thanks - that's very helpful
